core/jobs/batch_jobs/firebase_sync_jobs_test.py

```
@@ -0,0 +1,273 @@
+            uid='fb1', email='a@a.com', disabled=False
+        )
+
+        error = firebase_exceptions.InternalError('service down')
+        with self.firebase_sdk_stub.mock_delete_users_error(
+            batch_error_pattern=(error,)
+        ):
+            self.assert_job_output_is(
+                [
+                    job_run_result.JobRunResult(
+                        stderr=(
+                            'DELETED FAILURE: at slice=[0:1]: ' 'service down'
```

Ruff (0.15.15)

[warning] Implicitly concatenated string literals on one line

Combine string literals

(ISC001)